src/main/java/pl/kl/weatherservice/location/LocationsService.java

```diff
@@ -0,0 +1,67 @@
+package pl.kl.weatherservice.location;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import pl.kl.weatherservice.exceptions.EmptyInputException;
+import pl.kl.weatherservice.exceptions.InputOutOfRangeException;
+
+import java.util.Optional;
+import java.util.function.Predicate;
+
+@Component
+@RequiredArgsConstructor
+class LocationsService {
+
+    private final Double NORTH_LIMIT = 90.0;
+    private final Double SOUTH_LIMIT = -90.0;
+    private final Double EAST_LIMIT = 180.0;
+    private final Double WEST_LIMIT = -180.0;
+
+    private final LocationsRepository locationsRepository;
+
+    Location createLocation(String city, String region, String country, Double latitude, Double longitude) {
+        if (isStringEmpty(city)) {
```

Comment:
consider using `commons-lang3` library and `StringUtils.isBlank()` method

http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html
https://mvnrepository.com/artifact/org.apache.commons/commons-lang3

Reply:
Fixed.